src/serde_json.rs

```diff
@@ -0,0 +1,17 @@
+use bytes::{BufMut, BytesMut};
```

Comment:
Could this live in `src/serde.js` since that already exists and isn't too big?

Reply:
I was trying to mimic the `serde`/`serde_json` crates, but I suppose I could also go with a submodule to get `serde::json::to_bytes`?

Reply:
Just saw that we have a precedence with https://github.com/vectordotdev/vector/blob/a42e8bec54857a0b9b329b0b7a6b83edcc304f78/src/serde.rs#L41-L44, so just went ahead and followed your suggestion in f25d27c39270e736be68bcd6f9da1f6fc8152704.

Reply:
Works for me.